react/country/RUS.js

```diff
@@ -1,151 +1,148 @@
 import { ONE_LEVEL } from '../constants'
 import { firstLevelPostalCodes } from '../transforms/postalCodes'
-import {
-  getOneLevel,
-  getTwoLevels,
-} from '../transforms/addressFieldsOptions.js'
+import { getOneLevel } from '../transforms/addressFieldsOptions.js'
 
-const countryData = {
+const countryDataRaw = {
   'Республика Адыгея': {
-    'Maykop': '385000',
+    Maykop: '385000',
```

Comment:
Could you undo this formatting change? In the following lines as well
```suggestion
    'Maykop': '385000',
```

Just so we can keep a more precise PR change diff, and won't pollute the git blame as well

Reply:
I can do that, but this is prettier fixing the styles. I guess the previous PR haven't run `lint-staged` to match the prettier configuration. Let me know what you think.

Reply:
Ah, I see. In that case, there is no need to change. Thanks for linting it!